clicommand/bootstrap.go

```diff
@@ -454,7 +493,7 @@ var BootstrapCommand = cli.Command{
 			TracingServiceName:           cfg.TracingServiceName,
 			JobAPI:                       !cfg.NoJobAPI,
 			DisabledWarnings:             cfg.DisableWarningsFor,
-			KubernetesExec:               cfg.KubernetesExec,
+			KubernetesClient:             k8sClient,
```

Comment:
[no action required] Not relevant to this PR, but I got tripped by this "k8s client" naming a few times. To my brain it always means the "k8s api client". 

Agent socket client might be a better name?

Reply:
Great point, I'll come up with a better name in a minute...